app/controllers/staff/comments_controller.rb

```diff
@@ -0,0 +1,67 @@
+class Staff::CommentsController < Staff::BaseController
+  include Secured
+
+  def new
+    @activity = Activity.find(activity_id)
+    @report = Report.find(report_id)
+    @comment = Comment.new(activity_id: activity_id, report_id: report_id)
+    authorize @comment
+    @report_presenter = ReportPresenter.new(@report)
+  end
+
+  def create
+    @comment = Comment.create(comment_params)
+    authorize @comment
+    @comment.update(owner: current_user)
```

Comment:
I don't know if this is the same one or if I am misunderstanding what is happening here, but it looks like the `create` action is saving twice? Once with the `update` and then again with the `save!` - is that that the behavious we are after?

Could we just assign the owner to the new comment instance and then let the save! take care of things later?

Reply:
No you're right - my mistake!